editoast/src/views/v2/timetable/stdcm.rs

```diff
@@ -304,3 +398,8 @@ impl From<TrackOffsetExtractionError> for STDCMError {
         }
     }
 }
+
+enum MaxRunningTimeResult {
+    MaxRunningTime { value: u64 },
+    Error { error: Box<SimulationResponse> },
```

Comment:
I believe you don't need to `Box` it.

```suggestion
    Error { error: SimulationResponse },
```

Reply:
It's not strictly necessary but I had a linter warning about different memory size across the enum cases, the suggested fix was to box the larger one.

Reply:
Oh, I see, I believe we acknowledge that different variant size were ok, when the error path was the memory footprint (wasting some memory for a path that should not happen often and that is not a hot-path). In your case, it's the opposite, the error path is the big one, so I guess it's fine to leave it as a `Box`. I'm resolving.